prow/git/v2/interactor_test.go

```diff
@@ -472,44 +472,49 @@ func TestInteractor_BranchExists(t *testing.T) {
 	}
 }
 
-func TestInteractor_CommitExists(t *testing.T) {
+func TestInteractor_ObjectExists(t *testing.T) {
 	var testCases = []struct {
 		name          string
+		object        string
 		responses     map[string]execResponse
 		expectedCalls [][]string
 		expectedOut   bool
 		expectedErr   bool
 	}{
 		{
-			name: "happy case",
+			name:   "happy case",
+			object: "abc123",
 			responses: map[string]execResponse{
-				"branch --contains abc123": {out: []byte("")},
+				"cat-file -e abc123": {out: []byte("")},
 			},
 			expectedCalls: [][]string{
-				{"branch", "--contains", "abc123"},
+				{"cat-file", "-e", "abc123"},
 			},
 			expectedOut: true,
 		},
 		{
-			name: "Does not exist",
+			name:   "Does not exist",
+			object: "000000",
 			responses: map[string]execResponse{
-				"branch --contains abc123": {out: []byte(""), err: errors.New("error: no such commit abc123")},
+				"cat-file -e 000000": {out: []byte(""), err: nil},
 			},
 			expectedCalls: [][]string{
-				{"branch", "--contains", "abc123"},
+				{"cat-file", "-e", "000000"},
 			},
-			expectedOut: false,
+			expectedOut: true,
```

Comment:
This should be `false`, right? Same as below. 
I'd expect this to cause unit tests to fail.